R/function_defaults.R

```diff
@@ -0,0 +1,18 @@
+
+function_defaults <- function(fun.name, out=c('package','def.funs')){
+  funs <- list('points'=
```

Comment:
There is a list similar to this in config().  We may want to share this info with that  to reduce duplication.